bbb/servicebase.py

```diff
@@ -18,6 +19,18 @@
 ListenerServiceEvent = namedtuple("ListenerServiceEvent", ("exchange", "routing_key", "callback", "queue_name"))
 
 
+@contextmanager
+def lock_table(db, table_name):
+
+    try:
+        if "mysql" in db.url.get_backend_name():
+            db.execute(sa.text("LOCK TABLE {} WRITE;".format(table_name)))
```

Comment:
Yeah, SA won't let me use `:table` for table names. I get syntax errors. Tested using
```python
db.execute("SELECT * FROM :t", "tasks").fetchall()
```

Also http://stackoverflow.com/questions/13124572/binding-variable-to-table-name-with-cx-oracle tells me that I'm not the only one.